File: 1 Array.js
function upArray(arr){
  const fil1 = arr.length > 0;
    const fil2 = arr.every(s => s.toString().length === 1);
    if (fil1 && fil2) {
        const str1 = arr.join('');
        const str2 = (BigInt(str1) + BigInt(1)).toString().padStart(str1.length, '0');
        return Array.from(str2, Number);
    } else {
        return null;
    }
}

File: 1 Array.py
def up_array(arr):
    fil1 = len(arr)>0
    fil2 = all([len(str(s))==1 for s in arr])
    if fil1 and fil2:
        str1 = "".join(map(lambda x:str(x),arr))
        str2 = str(int(str1)+1).rjust(len(str1),'0')
        return list(map(lambda x:int(x),list(str2)))
    else:
        return None


